drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -126,16 +129,18 @@ public void setCopyFontType(String fontPath) {
 
         hasPaintChanged = true;
         requestLayout();
+        invalidate();
     }
 
-    public void setDropCapTextSize(@DimenRes float textSize) {
+    public void setDropCapTextSize(float textSize) {
```

Comment:
But aren't pixels indivisible? Anyway, suffixing the argument with `Px` is enough for me.